### ISSUE TITLE
Send app strings for translation

### ISSUE THREAD

**Issue by sylvia-sable** [theirc/coach-erevu#145]
Hi @secretrobotron and @ryanwarsaw do you need app strings translated for Tunakujenga (or new ones for Coach Erevu)? Let me know so I can get those translated ASAP.

**Comment by secretrobotron** [comment]
Yep. If there are any differences between the two, then we need some new strings translated.
For example, there is a message that's shown at the end of every quiz on Coach Erevu, which might not be the same for Tunakujenga. I'm not sure of what else there is that needs attention.
